RELEASE_PROCEDURE.md

```diff
@@ -11,31 +11,29 @@ When you release fixed version of SpotBugs, please follow these procedures.
 ## Release to Maven Central
 
 When we push a tag, the build result on GitHub Actions will be deployed to the [SonaType Nexus](https://oss.sonatype.org/), and published to the Maven Central automatically by [Gradle Nexus Publish Plugin](https://github.com/gradle-nexus/publish-plugin). Then we can find [artifacts in the Maven Central](https://repo1.maven.org/maven2/com/github/spotbugs/) after several hours.
+See the `build` phase in `.github/workflows/release.yml` calling the `publishToSonatype` and `closeSonatypeStagingRepository` gradle tasks.
 
 ## Release to Eclipse Update Site
 
-It's automated by Github CI.
+It's automated by GitHub CI.
 
 When we push tag, the build result will be deployed to [eclipse-candidate repository](https://github.com/spotbugs/eclipse-candidate).
 When we push tag and its name doesn't contain `_RC`, the build result will be deployed to [eclipse repository](https://github.com/spotbugs/eclipse).
 
-See `deploy` phase in `.github/workflows/release.yml` for detail.
+See `Deploy eclipse`, `Deploy eclipse-candidate` and `Deploy eclipse-latest` phases in `.github/workflows/release.yml` for detail.
 
 ## Release to Eclipse Marketplace
 
 Update version in [Eclipse Marketplace page](https://marketplace.eclipse.org/content/spotbugs-eclipse-plugin). If you have no permission, please contact with @KengoTODA or @iloveeclipse.
 
 ## Release to Gradle Plugin Portal
 
-No action necessary. When we push tag, the build result on Github CI will be deployed to Gradle Plugin Portal.
-
-See `deploy` phase in `.github/workflows/release.yml` for detail.
-
-## Update installation manual
```

Comment:
This part was modified in https://github.com/spotbugs/spotbugs/pull/451. There is no need to modify anything in connection with this.